test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorPagesTest.cs

```diff
@@ -167,6 +166,19 @@ public RazorPagesTest(MvcTestFixture<RazorPagesWebSite.Startup> fixture)
         }
 
         [Fact]
+        public async Task Page_Route_RouteData_StringToIntCoercion_NotFound()
```

Comment:
Sorry I still don't get what this is testing 😢 🐼 

Test names usually follow the pattern `SuchAndSuchFeature_InSuchAndSuchConditions_ExpectsSuchAndSuchResult`.

So, what component is being tested? Under what interesting setup/condition? And what is the expected result?